cpp/modules/luma.gl/garrow/src/array.cc

```diff
@@ -53,15 +53,29 @@ void Array::setData(const std::shared_ptr<arrow::Array>& data, wgpu::BufferUsage
     throw new std::runtime_error("Data with null values is currently not supported");
   }
 
-  auto buffers = data->data()->buffers;
+  // TODO(ilija@unfolded.ai): Not sure if this is the best way to extract the data out of any data type
+
   uint64_t offset = 0;
+  auto buffers = data->data()->buffers;
   // Starting from buffer at index 1, as the first buffer is used for null bitmap
   for (int i = 1; i < buffers.size(); i++) {
```

Comment:
I assume additonal buffers are used for indices into variable size lists, and since these cannot really be processed on the GPU, I assume we don't need to iterate over buffers.